target_info/target_info.lua

```diff
@@ -0,0 +1,82 @@
+local math = require('math')
+local string = require('string')
+local ui = require('ui')
+local target = require('target')
+
+local text_color = ui.color.lightgrey
+local config_state = {
+    title = 'Target Info',
+    style = 'chromeless',
+    x = 100,
+    y = 0,
+    width = 130,
+    height = 48,
+    color = ui.color.black,
+}
+function math.round(num, prec)
```

Comment:
Just define:
```lua
round = function(num, prec) ... end
```

No need to put it into the `math` table. Required tables should not be modified if it can be helped.